resources/s3_file.rb

```diff
@@ -21,9 +21,9 @@
 end
 
 # authentication
-property :aws_access_key, String
-property :aws_secret_access_key, String, sensitive: true
-property :aws_session_token, String, sensitive: true
+property :aws_access_key, String, sensitive: true
+property :aws_secret_access_key, String, sensitive: true, sensitive: true
```

Comment:
was already marked as as sensitive

Reply:
I disagree here - the access_key is the equivalent of a username.  I would consider usernames to be sensitive - would not want to post them on the internet.  It is useful to see them in logs, but since the consumer has control and visibility into the keys they are sending, they should be able to see how they are calling the provider without needing to defer to the logs for this.

Either way - the urgency is not to log the tokens for now.
I've reverted making all the access_keys sensitive for now, and fix the duplicate s3_file annotations.
@majormoses - I've made all the changes you've suggested.